string pickup
string dropoff
string task_id
float32 priority
